modules/notify/src/main/NotifyApi.scala

```diff
@@ -47,14 +47,13 @@ final class NotifyApi(
 
     def getAllows(userIds: Iterable[UserId], event: NotificationPref.Event): Fu[List[NotifyAllows]] =
       colls.pref.tempPrimary
-        .find($inIds(userIds), $doc(s"notification.${event.key}" -> true).some)
+        .find($inIds(userIds), $doc(event.key -> true).some)
         .cursor[Bdoc]()
         .listAll() dmap { docs =>
         for {
           doc    <- docs
           userId <- doc.getAsOpt[UserId]("_id")
-          allowsOpt = doc child "notification" flatMap (_ int event.key) map Allows.fromCode
-          allows    = allowsOpt | NotificationPref.default.allows(event)
+          allows = (doc int event.key).map(Allows.fromCode) | NotificationPref.default.allows(event)
```

Comment:
This is still a problem because defaults cannot be assigned to users without a preexisting document in notify_pref, there will be no find doc for them.  I left it alone for now as I was unsure if you planned to tackle this in a migration script (add notify_pref for every user) but if they're to be created lazily, we'll need additional logic.

Reply:
good catch, this should fix it https://github.com/lichess-org/lila/commit/a0790fa0282b2c8a59feafc4822bd3307671e73a